packages/itwinui-react/src/core/Tile/Tile.tsx

```diff
@@ -181,7 +181,6 @@ export const Tile = (props: TileProps) => {
         className,
       )}
       aria-disabled={isDisabled}
-      tabIndex={isActionable && !isDisabled ? 0 : undefined}
```

Comment:
This will break keyboard navigation for existing user code because (in your current version) it requires them to add `<LinkOverlay as='button'>`.

Should we automatically do that when `isActionable` is true? we can also handle disabled behavior.

Reply:
I think it should be good now. Adding isActionable class / prop.